src/github-graphql.js

```diff
@@ -271,9 +271,7 @@ class GraphQLClient {
         data.repository.pullRequest.comments.nodes);
     } else {
       this.warningCount++;
-      console.log('warning: no available reaction data!',
-        'owner:', owner, 'name:', name, 'pull request number:', number,
-        'error:', errors);
+      console.log('error:', errors);
```

Comment:
Hi, Instead of simply making the warning message shorter, maybe we can do something like this https://github.com/coala/gh-board/blob/ec19e220cc2ffe1c1f7dcf83c03ced2a189da039/src/github-graphql.js#L96
to make the error message configurable. This warning is usually because of GitHub API hits frequency limit, thus not very important.

Reply:
Please address this 😉

Reply:
Can you please explain this little bit, how to make the error message configurable ?

Reply:
Just use `if (DEBUG)` to wrap `console.log(xx)` around.

Reply:
What should be my console message, the earlier one ?

Reply:
> Hi, Instead of simply making the warning message shorter, maybe we can do something like this
> 
> [gh-board/src/github-graphql.js](https://github.com/coala/gh-board/blob/ec19e220cc2ffe1c1f7dcf83c03ced2a189da039/src/github-graphql.js#L96)
> 
> Line 96 in [ec19e22](/coala/gh-board/commit/ec19e220cc2ffe1c1f7dcf83c03ced2a189da039)
> 
>  if (DEBUG) { 
> 
> to make the error message configurable. This warning is usually because of GitHub API hits frequency limit, thus not very important.

I made the necessary changes. Please review this.